src/app/fdctl/configure/workspace.c

```diff
@@ -294,7 +301,8 @@ init( config_t * const config ) {
         cnc   ( pod, "cnc" );
         break;
       case wksp_forward:
-        cnc   ( pod, "cnc" );
+        cnc            ( pod, "cnc" );
+        leader_schedule( pod, "leader_schedule" );
```

Comment:
we need in both pack and forward right?

Reply:
I assume so we'll need to coordinate with @ptaffet-jump on that.